classes/01class/exercises/c01-aws01/gpavelar/README.md

```diff
@@ -0,0 +1,436 @@
+                  "GroupId": "sg-076a9212c8d1a8a45"
+                }
+              ],
+              "Ipv6Addresses": [],
+              "MacAddress": "02:1d:91:a2:16:0a",
+              "NetworkInterfaceId": "eni-09bfaa3a2bafcb736",
+              "OwnerId": "478433196210",
+              "PrivateDnsName": "ip-172-31-0-98.ap-southeast-2.compute.internal",
+              "PrivateIpAddress": "172.31.0.98",
+              "PrivateIpAddresses": [
+                {
+                  "Association": {
+                    "IpOwnerId": "amazon",
+                    "PublicDnsName": "ec2-3-25-65-76.ap-southeast-2.compute.amazonaws.com",
+                    "PublicIp": "3.25.65.76"
+                  },
+                  "Primary": true,
+                  "PrivateDnsName": "ip-172-31-0-98.ap-southeast-2.compute.internal",
+                  "PrivateIpAddress": "172.31.0.98"
+                }
+              ],
+              "SourceDestCheck": true,
+              "Status": "in-use",
+              "SubnetId": "subnet-351ced53",
+              "VpcId": "vpc-37d4dd50",
+              "InterfaceType": "interface"
+            }
+          ],
+          "RootDeviceName": "/dev/xvda",
+          "RootDeviceType": "ebs",
+          "SecurityGroups": [
+            {
+              "GroupName": "devops-develop",
+              "GroupId": "sg-076a9212c8d1a8a45"
+            }
+          ],
+          "SourceDestCheck": true,
+          "VirtualizationType": "hvm",
+          "CpuOptions": {
+            "CoreCount": 1,
+            "ThreadsPerCore": 1
+          },
+          "CapacityReservationSpecification": {
+            "CapacityReservationPreference": "open"
+          },
+          "HibernationOptions": {
+            "Configured": false
+          },
+          "MetadataOptions": {
+            "State": "applied",
+            "HttpTokens": "optional",
+            "HttpPutResponseHopLimit": 1,
+            "HttpEndpoint": "enabled"
+          }
+        }
+      ],
+      "OwnerId": "478433196210",
+      "ReservationId": "r-0dbe9c497a8460f09"
+    }
+  ]
+}
+```
+
+```bash
+# Connecting via SSH to this instance
+ssh -i /var/scratch/DevOps_Academy/DevOps-academy.pem  ec2-user@3.25.65.76
+# or
+ssh -i /var/scratch/DevOps_Academy/DevOps-academy.pem  ec2-user@ec2-3-25-65-76.ap-southeast-2.compute.amazonaws.com
+
+## Output
+The authenticity of host '3.25.65.76 (3.25.65.76)' can't be established.
+ECDSA key fingerprint is SHA256:Fv9yNqOFVQuU+wl8HKAR8cRV8o0wVHS9cJ4eq7HEqP8.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '3.25.65.76' (ECDSA) to the list of known hosts.
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+4 package(s) needed for security, out of 10 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-0-98 ~]$
+```
+
+Well, I think it is it. I don't know yet the best way to select an AMI.
```

Comment:
as we don't want this image ID hard-code,  they way you did using `resolve:ssm` is a good one, because you're looking up for a dynamic value somewhere else

Another way is using a bash script for looking up somewhere(Vault, SSM Parameter store, etc...) and just using  the output in the commands

Reply:
I got it! It would be a great improvement.